internal/ingress/controller/kong.go

```diff
@@ -1069,17 +1066,57 @@ func (n *NGINXController) syncUpstreams(locations []*ingress.Location, backends
 	return nil
 }
 
+func (n *NGINXController) syncCertificates(servers []*ingress.Server) error {
+
+	certsToKeep := make(map[string]bool)
+	for _, server := range servers {
+		if server.Hostname == "_" {
+			// there is no catch all server in kong
```

Comment:
does this worth a log?

Reply:
No, we just ignore them as this condition can't occur with Kong Ingress Controller.